magenta/models/coconet/lib_data.py

```diff
@@ -172,7 +172,7 @@ def __init__(self, **kwargs):
     """
     assert set(kwargs.keys()) == self.keys
     assert all(
-        len(value) == len(kwargs.values()[0]) for value in kwargs.values())
+        len(value) == len(list(kwargs.values())[0]) for value in list(kwargs.values()))
```

Comment:
To fix a line too long lint error:
```suggestion
        len(value) == len(list(kwargs.values())[0])
        for value in list(kwargs.values()))
```

Reply:
I implemented this and made a PR in @everettk 's repo. 
However, since it's not pulled yet, you could pull it from my repo instead, since I can't suggest an edit on this PR directly. 
Please advise if I should make a separate PR or how to proceed. :)